### ISSUE TITLE
Refactor to Python class

### ISSUE THREAD

**Issue by ivandokov** [ivandokov/phockup#28]
The code is using functions and there are a few with a lot of arguments in order to pass some global settings to the function that actually needs the argument. Such example is 
```
def handle_file(source_file, outputdir, dir_format, move_files, date_regex=None):
```
[here](https://github.com/ivandokov/phockup/blob/master/phockup.py#L225).

By refactoring to a class those global arguments could be class properties set by the constructor.

**Comment by mdujava** [comment]
Hello, 
I have time and I am willing to help with this enhancement.
Did you start on this task? Do you have some plans or some subtasks, that can be divided to multiple people?

**Comment by ivandokov** [comment]
I was planing to use the time that I have this week to start working on this issue since it is a priority one.
My plan is to separate the functionality to different classes because it will be easier to implement new stuff and test. For example the print on screen must be a separate class because it will be easier to display a progress bar (#21). Exif data access should be separate too. Files interaction (move/link/copy) too.

I started development on #20 but I got stuck. If you want you are more than welcome to help me there.
